pkg/inject/virtualgateway_envoy.go

```diff
@@ -127,12 +127,27 @@ func (m *virtualGatewayEnvoyConfig) getAugmentedMeshName() string {
 	return meshName
 }
 
+const (
+	// when enabled, a virtual gateway image will not be overriden
+	gatewayImageOverrideModeEnabled = "enabled"
```

Comment:
humm, seems this two const is not used..

Reply:
oops, my bad. A missed change. Fixed it. Thanks for catching it

Reply:
this is a little counter-intutive, maybe GatewaySkipImageOverrideMode?

Reply:
dang - of course 🤦  😄